sdk/spring/spring-cloud-azure-actuator-autoconfigure/src/main/java/com/azure/spring/cloud/actuate/autoconfigure/eventhubs/EventHubHealthConfiguration.java

```diff
@@ -23,12 +25,14 @@
 @Configuration
 @ConditionalOnClass({ EventHubClientBuilder.class, HealthIndicator.class })
 @AutoConfigureAfter(AzureEventHubAutoConfiguration.class)
+@ConditionalOnEnabledHealthIndicator("azure-eventhubs")
+@ConditionalOnProperty(value = "spring.cloud.azure.eventhubs.enabled", havingValue = "true", matchIfMissing = true)
+@ConditionalOnAnyProperty(prefix = "spring.cloud.azure.eventhubs", name = { "connection-string", "namespace" })
 public class EventHubHealthConfiguration {
 
     @Bean
     @ConditionalOnBean(EventHubClientBuilder.class)
```

Comment:
Is this condition not needed now? @ConditionalOnBean(EventHubClientBuilder.class)